### ISSUE TITLE
PCB for that integration

### ISSUE THREAD

**Issue by kamyk** [assembly12/Foxess-T-series-ESPHome-Home-Assistant#18]
Hello,

Since yesterday many people have a problems with an access to Foxcloud by HA. I have all parts and I would like to do it asap. @assembly12  do you maybe have that PCB to sell somewhere?

Thank you!
Kamyk

**Comment by assembly12** [comment]
Hi,
yes I have some spares lying around. If you live in Germany I can send you one. Otherwise it will probably be cheaper to order them staight from China ;-)
You can send my a pm over at the [HA Forum](https://community.home-assistant.io/u/assembly) if you like.

**Comment by kamyk** [comment]
> Hi, yes I have some spares lying around. If you live in Germany I can send you one. Otherwise it will probably be cheaper to order them staight from China ;-) You can send my a pm over at the [HA Forum](https://community.home-assistant.io/u/assembly) if you like.

Hi :) I live in Berlin :) We can do it! Let me know what should I send to you and how to communicate by mail.

Best,
Rafal

**Comment by kamyk** [comment]
> Hi, yes I have some spares lying around. If you live in Germany I can send you one. Otherwise it will probably be cheaper to order them staight from China ;-) You can send my a pm over at the [HA Forum](https://community.home-assistant.io/u/assembly) if you like.

Sent PM already. Thank you!

**Comment by spielkind1611** [comment]
@assembly12 
Do you perhaps still have a PCB lying around that you no longer need?

**Comment by assembly12** [comment]
Yeah, I've got some left. Since Private messaging isn't possible on github, please contact me on the Home Assistant forum.

**Comment by SibrenVasse** [comment]
I also have a few left, can ship them from the Netherlands or from Germany.

**Comment by spielkind1611** [comment]
How can i contact you?

**Comment by SibrenVasse** [comment]
My email is listed on my github profile.
